docs/source/tutorials/submission.ipynb

```diff
@@ -0,0 +1,233 @@
+{
```

Comment:
<p>Can we replace "you"  with "user/ the user / a user"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2874/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>